relay-common/Cargo.toml

```diff
@@ -21,7 +21,7 @@ log = { version = "0.4.8", features = ["serde"] }
 lru = "0.4.0"
 parking_lot = "0.10.0"
 regex = "1.3.9"
-sentry-types = "0.20.0"
+sentry-types = "0.14.1"
```

Comment:
It looks like there are quite a few unintended changes to build and lock files. Did a merge go wrong?

Reply:
It is reverted.